zio-http/src/test/scala/zhttp/middleware/MiddlewareSpec.scala

```diff
@@ -50,13 +50,13 @@ object MiddlewareSpec extends DefaultRunnableSpec with HttpAppTestExtensions {
         assertM(program)(equalTo(Vector("200 GET /health 1000ms\n")))
       } +
         testM("log 404 status method url and time") {
-          val program = run(Http.empty @@ debug.withEmpty) *> TestConsole.output
+          val program = run(Http.empty.as(Response.ok) @@ debug.withEmpty) *> TestConsole.output
```

Comment:
this is a breaking change. Is there a way we can support `Http.empty @@ debug.withEmpty` as well?

Reply:
Agree. Need to think more about this. Ideally it should get fixed with #616.